snmp/datadog_checks/snmp/data/profiles/palo-alto.yaml

```diff
@@ -63,4 +63,56 @@ metrics:
     symbol:
       OID: 1.3.6.1.4.1.25461.2.1.2.5.1.3
       name: panGPGWUtilizationActiveTunnels
-
+  - MIB: PAN-ENTITY-EXT-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.25461.1.1.7.1.1.2
+      name: panEntityTotalPowerUsed
+    # metric_tags:
+    #   - MIB: ENTITY-MIB
+    #     column:
+    #       OID: 1.3.6.1.2.1.47.1.2.1.1.2
+    #       name: entLogicalDescr
+    #     table: entLogicalTable
+    #     tag: ent_descr
+  - MIB: PAN-ENTITY-EXT-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.25461.1.1.7.1.1.1
+      name: panEntityTotalPowerAvail
+    # metric_tags:
+    #   - MIB: ENTITY-MIB
+    #     column:
+    #       OID: 1.3.6.1.2.1.47.1.2.1.1.2
+    #       name: entLogicalDescr
+    #     table: entLogicalTable
+    #     tag: ent_descr
+  - MIB: PAN-ENTITY-EXT-MIB
+    table:
+      OID: 1.3.6.1.4.1.25461.1.1.7.1.2.1
+      name: panEntityFRUModuleTable
+    symbols:
+      - OID: 1.3.6.1.4.1.25461.1.1.7.1.2.1.1.1
+        name: panEntryFRUModulePowerUsed
+      - OID: 1.3.6.1.4.1.25461.1.1.7.1.2.1.1.2
+        name: panEntryFRUModuleNumPorts
+    metric_tags:
+      - MIB: ENTITY-MIB
+        column:
+          OID: 1.3.6.1.2.1.47.1.1.1.1.16
+          name: entPhysicalIsFRU
+        table: entPhysicalTable
+        tag: fru
```

Comment:
I think we're missing a tag here, and this tag is not correctly named. This is a boolean, we want an index or a name.